### ISSUE TITLE
1.47 fails to build with musl due to use of `uint`

### ISSUE THREAD

**Issue by ncopa** [hreinecke/sg3_utils#77]
Build fails on alpine linux with the following error:
```
gcc -DHAVE_CONFIG_H -I. -I..  -iquote ../include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64  -Os -fomit-frame-pointer -Wall -W  -Os -fomit-frame-pointer -MT sg_dd.o -MD -MP -MF .deps/sg_dd.Tpo -c -o sg_dd.o sg_dd.c
sg_dd.c: In function 'main':
sg_dd.c:2402:17: error: unknown type name 'uint'; did you mean 'int'?
 2402 |                 uint off;
      |                 ^~~~
      |                 int
make[2]: *** [Makefile:1172: sg_dd.o] Error 1
```

**Comment by ncopa** [comment]
Fix is trivial:
```diff
diff --git a/src/sg_dd.c b/src/sg_dd.c
index 9d05c93..7662728 100644
--- a/src/sg_dd.c
+++ b/src/sg_dd.c
@@ -2399,7 +2399,7 @@ main(int argc, char * argv[])
             res = blocks * blk_sz;
             if (iflag.zero && iflag.ff && (blk_sz >= 4)) {
                 uint32_t pos = (uint32_t)skip;
-                uint off;
+                unsigned int off;
 
                 for (k = 0, off = 0; k < blocks; ++k, off += blk_sz, ++pos) {
                     for (j = 0; j < (blk_sz - 3); j += 4)
```

**Comment by doug-gilbert** [comment]
This has been fixed in my upstream version. See:
https://github.com/doug-gilbert/sg3_utils

Thanks.

**Comment by hreinecke** [comment]
Fixed with 1.48.
